### ISSUE TITLE
PhaseQuadMuon improvements

### ISSUE THREAD

**Issue by raquelalvarezbanos** [mantidproject/mantid#11588]
This issue was originally [TRAC 10746](http://trac.mantidproject.org/mantid/ticket/10746)

Some minor improvements:
- Remove conversion from micro secs to nano secs
- Use blocksize() when possible
- Add Y axis label in output workspace

After discussion with muon scientists, we agreed that:
- PulseOver, MeanLag and DoublePulse options should be removed from the algorithm. The first two options are used to set Y errors from [0, PulseOver] to big numbers, effectively disabling the corresponding data. DoublePulse
- DeadTimeCorrections shouldn't be applied here. It makes more sense to run ApplyDeadTimeCorr to the data and follow that with PhaseQuad, so it no longer needs to consider dead times.
- The same for Efficiency. We talked about the possibility of creating a new algorithm, but we need to discuss this in more detail.
- Detector phases should be calibrated in a separate algorithm (see issues #11868 and #11869)

In summary, the new version of PhaseQuad should take an input workspace, a Table workspace (containing the detector phases) and produce an output workspace.
